### PR TITLE
Issue #9581: updated example of AST for TokenTypes.TEXT_BLOCK_CONTENT

### LINKED ISSUE #9581: update example of AST for TokenTypes.TEXT_BLOCK_CONTENT
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#TEXT_BLOCK_CONTENT

needs to be extended to show example of AST.
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
Resolves #9581 
erfsdfsdf

```
public class MyClass {
    String example = """
            Hello, World!
            """;
}
java -jar checkstyle-8.41-all.jar -t Test.java
CLASS_DEF -> CLASS_DEF [3:0]
|--MODIFIERS -> MODIFIERS [3:0]
|   `--LITERAL_PUBLIC -> public [3:0]
|--LITERAL_CLASS -> class [3:7]
|--IDENT -> MyClass [3:13]
`--OBJBLOCK -> OBJBLOCK [3:21]
    |--LCURLY -> { [3:21]
    |--VARIABLE_DEF -> VARIABLE_DEF [4:4]
    |   |--MODIFIERS -> MODIFIERS [4:4]
    |   |--TYPE -> TYPE [4:4]
    |   |   `--IDENT -> String [4:4]
    |   |--IDENT -> example [4:11]
    |   |--ASSIGN -> = [4:19]
    |   |   `--EXPR -> EXPR [4:21]
    |   |       `--TEXT_BLOCK_LITERAL_BEGIN -> """ [4:21]
    |   |           |--TEXT_BLOCK_CONTENT -> \n            Hello, World!\n             [4:24]
    |   |           `--TEXT_BLOCK_LITERAL_END -> """ [6:12]
    |   `--SEMI -> ; [6:15]
    `--RCURLY -> } [7:0]


```

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -3849,23 +3849,34 @@
      *
      * <p>For example:</p>
      * <pre>
-     *         String hello = """
-     *                 Hello, world!
-     *                 """;
+     * public class MyClass {
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:3852]: You can use old example, but use corrected subtree representation in `parses as:` section

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (Vyom-Yadav) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4133]: done

Comment (timurt) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4133]: Not done, I meant use old example without class definition
```
     *         String hello = """
     *                 Hello, world!
     *                 """;
```

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 2]
```diff
@@ -4130,23 +4130,34 @@
...
-     * |   |           `--TEXT_BLOCK_LITERAL_END (""")
-     * |   `--SEMI (;)
+     * CLASS_DEF
+     * |--MODIFIERS
+     * |   `--LITERAL_PUBLIC (public)
+     * |--LITERAL_CLASS (class)
+     * |--IDENT (MyClass)
+     * `--OBJBLOCK
+     *     |--LCURLY ({)
+     *     |--VARIABLE_DEF
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4149]: should be `VARIABLE_DEF -&gt; VARIABLE_DEF` please correct all subtree

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (Vyom-Yadav) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java]: done

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 2 above)
Comment (Vyom-Yadav) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java]: done

**Review by timurt — APPROVED** [review]

**Review by romani — APPROVED** [review]

**Pull request merged**